validations/src/validations.rs

```diff
@@ -293,7 +295,15 @@ pub fn validate_mint_transaction(
 /// Function to validate a rad request
 pub fn validate_rad_request(rad_request: &RADRequest) -> Result<(), failure::Error> {
     let retrieval_paths = &rad_request.retrieve;
+    // If the data requests has no retrive it is set as invalid
+    if retrieval_paths.is_empty() {
+        return Err(RadError::UnsupportedReducerInAT { operator: 0 }.into());
```

Comment:
Isn't this supposed to be `NoRetrievalSource`?

Reply:
This error already existed for the case of empty retrieve vector, however the `NoRetrivalSource` is the error for the specific case in which there is no url parameter in the retrieve vector

Reply:
This error happens in that case, but it should be only for unsupported reducers, so as @aesedepece says, we could return the same error (aka NoRetrievalSources)